modules/porous_flow/doc/content/modules/porous_flow/sinks.md

```diff
@@ -29,6 +29,19 @@ Such example is shown here:
 
 Note that the parameter ```mass_flux``` is positive for a source and negative for a sink.
 
+
+### Injecting fluid at specified temperature
+
+When injecting a fluid at specified temperature (also computed as a postprocessor), users have to add another Dirac
+kernel [`PorousFlowPointEnthalpySourceFromPostprocessor`](/PorousFlowPointEnthalpySourceFromPostprocessor.md)
```

Comment:
I think you should replace "users have to add" by "users can add".  Then after the sentence "Users can also fix the temperature of an injected fluid using a Dirichlet BC, but this adds/subtracts heat energy to entire nodal volumes of porous material and fluid, so may lead to unacceptable errors from the additional heat energies added/removed"